widgets/partials/admin-monk-language-switcher.php

```diff
@@ -0,0 +1,27 @@
+<?php
+/**
+ * Monk Language Switcher.
+ *
+ * @link       https://github.com/brenoalvs/monk
+ * @since      1.0.0
+ *
+ * @package    Monk
+ * @subpackage Monk/Widgets/Partials
+ */
+
+$title = ! empty( $instance['title'] ) ? $instance['title'] : __( 'New title', 'text_domain' );
```

Comment:
Every localized text using translation functions as `_e` or `__` needs a second parameter called `text-domain`.

Check how these functions works here: https://codex.wordpress.org/I18n_for_WordPress_Developers

You can set all text-domains to `monk` for now.